zip-1014.rst

```diff
@@ -326,24 +337,24 @@ Panel as input into the Major Grant Review Committee which governs the
 It is highly desirable to develop robust means of decentralized community
 voting and governance –either by expanding the Community Panel or a
 successor mechanism– and to integrate them into this process by the end of
-2021. ECC and ZF SHOULD place high priority on such development and its
+2021. BO, ECC, and ZF SHOULD place high priority on such development and its
 deployment, in their activities and grant selection.
 
 
 ZF Board Composition
 --------------------
 
-Members of ZF's Board of Directors MUST NOT hold equity in ECC or have current
-business or employment relationships with ECC, except as provided for by the
-grace period described below.
+Members of ZF's Board of Directors MUST NOT hold equity in ECC or
+have current business or employment relationships with ECC, except as provided
+for by the grace period described below.
```

Comment:
Revert this formatting change (there is no semantic change to this paragraph, so it's confusing to keep it in the diff).